src/web/js/beforePyret.js

```diff
@@ -188,20 +188,24 @@ $(function() {
     const mac = CodeMirror.keyMap.default === CodeMirror.keyMap.macDefault;
     const modifier = mac ? "Cmd" : "Ctrl";
 
-    var cmOptions = {
-      extraKeys: CodeMirror.normalizeKeyMap({
+    defaultKeyMap = CodeMirror.normalizeKeyMap({
         "Shift-Enter": function(cm) { runFun(cm.getValue()); },
         "Shift-Ctrl-Enter": function(cm) { runFun(cm.getValue()); },
         "Tab": "indentAuto",
         "Ctrl-I": reindentAllLines,
-        "Esc Left": "goBackwardSexp",
         "Alt-Left": "goBackwardSexp",
-        "Esc Right": "goForwardSexp",
         "Alt-Right": "goForwardSexp",
         "Ctrl-Left": "goBackwardToken",
         "Ctrl-Right": "goForwardToken",
         [`${modifier}-/`]: "toggleComment",
-      }),
+    });
+    CPO.noVimKeyMap = CodeMirror.normalizeKeyMap({
```

Comment:
Please let me know if there is a better place to store the non-vim specific keymap than in `CPO` :)